### PR TITLE
fix: mobile font and vscode install instructions

### PR DESCRIPTION
**Pull request by greghroberts**
# Title

Fixed font issues on mobile devices and updated VSCode Editor Installation

## Description

* fix: mobile font issues on hero text
* fix: vscode install instructions now that extension is published
* fix: markdown format issues on quickstart
* fix: cloud optimizations 👍 

## Testing

Verified locally with dev server

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## License

- [ x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa-games/litexa/issues
[license]: https://www.apache.org/licenses/LICENSE-2.0
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: docs/.vuepress/theme/styles/fonts.styl [hunk 1]
```diff
@@ -18,3 +18,8 @@
     font-family: 'Amazon Ember';
     src: url('../fonts/AmazonEmber_Rg.ttf');
     font-weight: 400;
+
+@font-face
+    font-family: 'Amazon Ember';
+    src: url('../fonts/AmazonEmber_Md.ttf');
+    font-weight: 600
```
Comment (lvalencia) [inline docs/.vuepress/theme/styles/fonts.styl]: You're missing a semicolon and new line ending here.

File: docs/book/appendix-editor-support.md [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+![Visual Studio Code Install Prompt](../assets/images/vscode-litexa-extension.png)
+
+1. Click on "Search Marketplace"
+1. It should find the extension called "Litexa Syntax Highlighting", click "Install"
+1. That's it!
+
+### Alternative Install
+
+You can also search the Visual Studio Marketplace within Visual Studio Code for `ext:litexa` or `litexa` and install from there. More info about the extension can be found here:
+[https://marketplace.visualstudio.com/items?itemName=litexa.litexa-syntax-highlighting](https://marketplace.visualstudio.com/items?itemName=litexa.litexa-syntax-highlighting)
```
Comment (joseifert) [inline docs/book/appendix-editor-support.md]: Would potentially just add this as a footnote to the above "Installation", as opposed to having the separate header.

`Alternatively, you can also search the Visual Studio Marketplace ...`

File: docs/.vuepress/theme/styles/fonts.styl (hunk 1 above)
Comment (greghroberts) [reply docs/.vuepress/theme/styles/fonts.styl]: good catch. will update

File: docs/book/appendix-editor-support.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+
+## Manual Installation from Source
+
 To install this extension, you will need to have cloned the [alexa-games/litexa](https://github.com/alexa-games/litexa)
```
Comment (joseifert) [inline docs/book/appendix-editor-support.md]: Would potentially change the wording to something similar to the current VS Code README:

```
## Contribution & Customization

If you'd like to contribute to the extension or customize it for your own usage, you can download it from
[here](https://github.com/alexa-games/litexa/tree/master/packages/litexa-vscode/litexa) and install it locally. For
instructions on how to install a local VS Code extension, please refer to the documentation
[here](https://vscode-docs.readthedocs.io/en/stable/extensions/example-hello-world/#installing-your-extension-locally).
```

**Pull request merged**